### ISSUE TITLE
Unable to use remote connection dialect

### ISSUE THREAD

**Issue by tim37021** [pureqml/qmlcore#164]
To reproduce..
```qml
Rectangle {
  id: btn1;
  ClickMixin {}
}
XXX {
  btn1.onClicked: {

  }
}
```
in qml.app.js:
**_get('btn1')**.on('click')
> ReferenceError: _get is not defined

should be context._get('btn1')

**Comment by whoozle** [comment]
ah, I see, this is working for property, but not for IDs, I'll fix it shortly

**Comment by whoozle** [comment]
fixed #30 in d86b56d77b3390c3e22d762cc983d689e74eaaf4
```
Item {
Rectangle {
  id: btn1;
  ClickMixin {}
  width: 100;
  height: 100;
  color: "#030";
}
Item {
  btn1.onClicked: {
    log("CLICKED")
  }
}
}
```
is working fine

**Comment by tim37021** [comment]
Hi @whoozle 
Please reopen this issue, it still has error..

this code works
```qml
    Connection {
        target: context;
        onWidthChanged: {
            log('yoyoyo')
        }
    }
```
the following doesnot work(I put it in app.qml)
```qml
context.onWidthChanged: {
}
```
_context is not defined.....

**Comment by whoozle** [comment]
ah, I see. that's because `context` and `parent` are specially optimised, sorry about that, I'll try to fix it asap

**Comment by whoozle** [comment]
context.onWidthChanged fixed in cca410b
